magicctapipe/io/io.py

```diff
@@ -798,14 +623,14 @@ def load_train_data_files_tel(
     input_dir : str
         Path to a directory where input DL1-stereo files are stored
     config : dict
-        Yaml file with information about the telescope IDs.
-    offaxis_min : str, optional
+        Dictionary with information about the telescope IDs.
+    offaxis_min : str
         Minimum shower off-axis angle allowed, whose format should be
         acceptable by `astropy.units.quantity.Quantity`
-    offaxis_max : str, optional
+    offaxis_max : str
```

Comment:
but the old behaviour None= no cut is conserved, right? If you do not write that it is optional, than maybe write it in the description below

Reply:
Now I see it is again set as optional.